src/listr/tasks/setup/setupLocalPresetTaskFactory.js

```diff
@@ -0,0 +1,427 @@
+        task: async (ctx, task) => {
+          ctx.nodeCount = await task.prompt({
+            type: 'Numeral',
+            message: 'Enter the number of masternodes',
+            initial: 3,
+            float: false,
+            min: 3,
+            max: 6,
+            validate: (state) => {
+              if (+state < 3 || +state > 6) {
+                return 'You must set from 3 up to 6 nodes';
+              }
+
+              return true;
+            },
+          });
+        },
+      },
+      {
+        // hidden task to dynamically get
+        // host.docker.internal ip address
+        task: async (ctx) => {
+          // pulling image before run
+          // or we can get 404 error (on first run)
+          await new Promise((resolve, reject) => {
+            docker.pull('alpine', (err, stream) => {
+              if (err) {
+                reject(new Error(`Can't pull alpine image: ${err.message}`));
+
+                return;
+              }
+
+              const onProgress = () => {};
+
+              const onFinished = (error) => {
+                if (!error) {
+                  resolve();
+                } else {
+                  reject(new Error(`Can't pull alpine image: ${error.message}`));
+                }
+              };
+
+              docker.modem.followProgress(stream, onFinished, onProgress);
+            });
+          });
+
+          const platform = os.platform();
+
+          const hostConfig = {
+            AutoRemove: true,
+          };
+
+          if (platform !== 'darwin' && platform !== 'win32') {
+            hostConfig.ExtraHosts = ['host.docker.internal:host-gateway'];
+          }
+
+          const writableStream = new WritableStream();
+
+          const [result] = await docker.run(
+            'alpine',
+            [],
+            writableStream,
+            {
+              Entrypoint: ['sh', '-c', 'ping -c1 host.docker.internal | sed -nE \'s/^PING[^(]+\\(([^)]+)\\).*/\\1/p\''],
+              HostConfig: hostConfig,
+            },
+          );
+
+          const output = writableStream.toString();
+
+          if (result.StatusCode !== 0) {
+            throw new Error(`Can't get host.docker.internal IP address: ${output}`);
+          }
+
+          ctx.hostDockerInternalIp = output.trim();
+        },
+      },
+      {
+        // hidden task to dynamically create
+        // multinode tasks listr
+        // we need to add one more node (number of masternodes + 1) which controls masternodes
+        task: (ctx) => {
+          const subTasks = [];
+
+          for (let i = 0; i <= ctx.nodeCount; i++) {
+            const configReference = `config_${i + 1}`;
+
+            subTasks.push({
+              title: `Setup node #${i + 1}`,
+              task: () => new Listr([
+                {
+                  title: 'Create config',
+                  task: () => {
+                    const configName = `${ctx.preset}_${i + 1}`;
```

Comment:
We shouldn't count seed node as a MN. So with 2 nodes setup it should 2 MNs and 1 seed node. Also, it make sense to name config differently (i.e. `local_seed`)